packages/generate/src/gel-prisma.ts

```diff
@@ -0,0 +1,714 @@
+import type { CommandOptions } from "./commandutil";
+import type { Client } from "edgedb";
+import * as fs from "node:fs";
+
+const WARNINGS: string[] = [];
```

Comment:
Should we log these?

Reply:
Maybe instead of collecting them, just print them into stderr as they happen?

Reply:
I think you can print them to stdout with `console.log`, tbh. I don't think our debug output in our generators are very principled about that sort of thing.